tests/framework/PostProcessors/InterfacedPostProcessor/test_HS2PS.xml

```diff
@@ -92,30 +87,6 @@
       <type>csv</type>
       <source>outMC_PP</source>
     </Print>
-    <Plot  name="plot1" overwrite="False">
-      <plotSettings>
-        <plot>
-          <type>line</type>
-          <interpolationType>cubic</interpolationType>
-          <interpPointsX>100</interpPointsX>
-          <x>outMC|Output|time</x>
-          <y>outMC|Output|x</y>
-        </plot>
-        <plot>
-          <type>scatter</type>
-          <x>outMC_PP|Output|time</x>
-          <y>outMC_PP|Output|x</y>
-        </plot>
-        <xlabel>time</xlabel>
-        <ylabel>x</ylabel>
-      </plotSettings>
-      <actions>
-        <how>pdf</how>
-        <title>
-          <text>Plot History</text>
-        </title>
-      </actions>
-    </Plot>
```

Comment:
why plot is removed?

Reply:
I removed since it was not part of the testing capability